src/test/java/cucumber/steps/StepDefinitions.java

```diff
@@ -0,0 +1,45 @@
+package cucumber.steps;
+
+import controllers.DashboardController;
+import controllers.LaunchController;
+import io.cucumber.java.en.When;
+
+import java.util.ArrayList;
+
+import static helpers.UserTokenGenerator.getToken;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasItem;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class StepDefinitions {
+    private LaunchController launch = new LaunchController(Hooks.api);
+    private DashboardController dashboards = new DashboardController(Hooks.api);
+    private ArrayList<String> listWithDashboardNames;
+
+    @When("user is logged in")
+    public void checkUserIsLoggedIn() {
+        assertNotNull(getToken(), "Authorization failed");
+    }
+
+    @When("user makes request to get all permitted dashboard resources for the project")
+    public void getAllPermittedDashboardResourcesForTheProject() {
+        listWithDashboardNames = dashboards.getAllDashboardNames();
+    }
+
+    @When("user sees dashboard with {string} name")
+    public void checkDashboardNameIsInList(String dashboardName) {
+        assertThat(listWithDashboardNames, hasItem(dashboardName));
+    }
+
+    @When("user makes request to get info of launch with id {int}")
+    public void userMakesGetRequestToGetInfoOfLaunchWithId(int launchId) {
+        launch.getExactLaunchInfo(launchId);
+    }
+
+    @When("user sees that parameter {string} has value {int}")
```

Comment:
Maybe use ```@Then``` for verifications?

Reply:
But for what? It is steps file, we may use only the one annotation to reduce the import. Another annotations for feature files

Reply:
For better clarity.
[Then steps are used to describe an expected outcome, or result.](https://cucumber.io/docs/gherkin/reference/)

And why should we reduce imports? It doesn't reduce build dependencies, it only clarifies purpose of the step using appropriate annotation.

Reply:
For better clarity.
[Then steps are used to describe an expected outcome, or result.](https://cucumber.io/docs/gherkin/reference/)

And why should we reduce imports? It doesn't reduce build dependencies, it only clarifies purpose of the step using appropriate annotation.

Reply:
According to my experience in writing test with Cucumber, it worth to pay attention to annotations in features only. Cause feature-files are for bussiness initially. Steps files are for developers, and annotation is not so necessary. But maybe I had bad practice :)

Reply:
You are developer in test automation, please keep layers coherent.

Reply:
update annotations